src/fireEvent.js

```diff
@@ -119,11 +123,38 @@ const invokeEvent = (
   return returnValue;
 };
 
+const tryInvokeEvent = (...args) => {
+  try {
+    return invokeEvent(...args);
```

Comment:
We actually don't the `invokeEvent` to be throwable. It supposed to be there as a debugging aid, but the original Testing Library don't have this behavior and here it actually complicates things a little more. 

How about removing this behavior altogether in a separate PR and adjusting this one to keep the `invokeEvent` without the try/catch?

Reply:
Sure thing!
Just to make sure i understood 🤓 
- create a PR that removes the throwing code from `invokeEvent`
- remove the tryInvokeEvent from this PR

Reply:
Yup :)

Reply:
Here it is: #691